gtnh-modpack.json

```diff
@@ -1772,7 +1772,7 @@
       "filename": "NodalMechanics-1.7.10-1.1.-6-GTNH.jar",
       "download_url": "https://api.github.com/repos/GTNewHorizons/Nodal-Mechanics/releases/assets/52235570",
       "browser_download_url": "https://github.com/GTNewHorizons/Nodal-Mechanics/releases/download/1.1.-6-GTNH/NodalMechanics-1.7.10-1.1.-6-GTNH.jar",
-      "license": "Custom license, code must be open source",
+      "license": "Custom License, code must be open source, no derivative license specified.",
```

Comment:
Are you sure this properly represents the "license" stated on the [original repo](https://github.com/AKA-Syenite/Nodal-Mechanics#license)?

Reply:
The license says:

You are free to do whatever you wish with this mod and this source code, provided that you provide proper attribution. Please don't distribute this mod behind any sort of ad-wall such as adf.ly or the like. Please make any modifications of this code open-source.

We have adhered to the license, technically, but there is no open-source license for the derivative code on the GTNH [repository](https://github.com/GTNewHorizons/Nodal-Mechanics) listed.

Reply:
Ah, I missunderstood: I thought it meant that the license forbids derivations, not that there is no text about derivations.